src/cmd-remote-prune

```diff
@@ -29,24 +29,30 @@ builds = Builds(args.workdir)
 scanned_builds = []
 for build in builds.raw()["builds"]:
     for arch in build['arches']:
-        build = fetch_build_meta(build['id'], arch)
+        build = fetch_build_meta(builds, build['id'], arch, args.bucket, args.prefix)
         if build:
             scanned_builds.append(build)
 
 new_builds = []
 builds_to_delete = []
 
 # Find unreferenced builds in the bucket and remove them
-buildids = [x['id'] for x in scanned_builds]
+buildids = [x.id for x in scanned_builds]
 unreferenced_s3_builds = get_unreferenced_s3_builds(buildids, args.bucket, args.prefix)
 
+if args.dry_run:
+    print("Not removing anything: in dry-run mode")
+    sys.exit(0)
```

Comment:
nit: `raise SystemExit(0)` :smile:

Reply:
Done, one less imprort!

Reply:
I know those are equivalent, though... `sys.exit(0)` in the "top-level" code seems more idiomatic to me (whereas I could see `raise SystemExit(0)` being more of a library thing, though libraries ideally shouldn't ever do this).

Anyway, not strongly against, though I'm probably going to keep using the `sys.exit(main())` pattern. :smile:

Reply:
Reverted this back to `sys.exit`, which is being used in end-user commands more often